config/routes.rb

```diff
@@ -1,17 +1,16 @@
 Rails.application.routes.draw do
-  
+
   get 'password_resets/new'
   get 'password_resets/edit'
   get 'profiles/new'
   get 'residents/new'
-  
+
   get 'home', to: 'static_pages#home'
   get 'about', to: 'static_pages#about'
   get 'help', to: 'static_pages#help'
-  get 'directory', to: 'static_pages#directory'
   get 'display', to: 'static_pages#display'
-  get 'pictures', to: 'profiles#pictures'
-  
+  get 'directory', to: 'profiles#directory'
+
   resources :profiles do
```

Comment:
Since you've removed the `index` action, you can remove the corresponding route by changing this to `resources :profiles, except: [:index] do …`. The biggest benefit of this is it won't confuse any developers in the future.

Reply:
@iangreenleaf What do you think about making the directory page the index action of `/profiles`?  It feels more RESTful to me.

Reply:
Yep, I think that would be a great way to do it! The directory page *is* effectively your index page, with some optional filtering applied.